terraform/sqs_lambda/sqs_lambda.tf

```diff
@@ -1,23 +1,23 @@
 module "sqs_lambda" {
-  source                    = "git::https://github.com/cloudmitigator/reflex-engine.git//modules/sqs_lambda?ref=v1.0.0"
+  source                    = "git::https://github.com/cloudmitigator/reflex-engine.git//modules/sqs_lambda?ref=v2.0.0"
   cloudwatch_event_rule_id  = var.cloudwatch_event_rule_id
   cloudwatch_event_rule_arn = var.cloudwatch_event_rule_arn
-  function_name   = "AccountPasswordPolicyInsecureRule"
-  source_code_dir = "${path.module}/../../source"
-  handler         = "reflex_aws_account_password_policy_insecure.lambda_handler"
-  lambda_runtime  = "python3.7"
+  function_name             = "AccountPasswordPolicyInsecureRule"
+  package_location          = "${var.package_location}"
```

Comment:
We should switch to just var.package_location as the interpolation is not necessary